docs/index.qmd

```diff
@@ -40,6 +41,40 @@ about:
 ::: {#about}
 :::
 
+
+## Try it out!
+
+
+```{python}
+#| echo: false
+#| output: asis
+from urllib.parse import urlencode
+
+lines = """
+%pip install numpy pandas tzdata
+import pyodide_js
+await pyodide_js.loadPackage("https://storage.googleapis.com/ibis-wasm-wheels/pyarrow-16.0.0.dev2661%2Bg9bddb87fd-cp311-cp311-emscripten_3_1_46_wasm32.whl")
```

Comment:
This can ideally be removed when there's a pyodide distribution that ships with pyarrow.